Sources/StreamChatUI/ChatChannel/Attachments/ChatMessageGiphyView.swift

```diff
@@ -43,6 +43,12 @@ open class ChatMessageGiphyView<ExtraData: ExtraDataTypes>: View, UIConfigProvid
     deinit {
         imageTask?.cancel()
     }
+    
+    override open func setUp() {
+        super.setUp()
+        
+        imageView.contentMode = .scaleAspectFill
```

Comment:
This is very hard to decide but I would say this belongs to `setupAppearance` but it's not defined... Open to discussion, you can let it be...

Reply:
@DominikBucher12 you meant `defatultAppearance()` right? `setUpAppearance()` is for subclasses.

Reply:
Yep, checked that in other places we are treating this like appearance and fixed it

Reply:
Indeed, `defaultAppearance` it is.